src/replaceCustomValues.ts

```diff
@@ -404,7 +404,7 @@ export function replaceInjectablesWithSubmissionValues(
   })
   return CUSTOM_VALUES.reduce((newString, customValue) => {
     return newString.replace(
```

Comment:
Thoughts on using replaceAll here with the string?

Reply:
Ahh, Thought that should have worked at first, but didn't come up as a function. Need to change compiler to target `es2021` if we want to use it.

Reply:
Or later.

Reply:
I haven't really thought too much about our target runtimes for SDKs. Prob not a drama if we want to update that.

Reply:
nah, lets think about another time, merging!